package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "dictionary-component",
+  "version": "1.0.0",
+  "description": "Dictionary Component for CSV lookups on the eio platform",
+  "author": "elastic.io GmbH",
+  "scripts": {
+    "test": "mocha spec --recursive"
```

Comment:
Need to add eslint starter script `"pretest": "eslint lib/ spec verifyCredentials.js --fix",`